app/assets/stylesheets/comments.scss

```diff
@@ -44,6 +44,29 @@
     }
   }
 
+  .permalink {
+    @include transition(opacity);
+    opacity: 0;
+
+    [class^="entypo-"],
+    [class*="entypo-"] {
+      color: $text-color-pale;
+    }
+  }
+
+  .comment:hover .permalink {
+    opacity: .8;
+
+    &:hover {
+      opacity: 1;
+
+      [class^="entypo-"],
```

Comment:
- Nesting should be no greater than 3, but was 4
- Selector should have depth of applicability no greater than 3, but was 4

Reply:
I tried to fix this one but it wasn't possible without changing many things. I feel it's okay to merge it like that.